pxtarget.json

```diff
@@ -497,6 +497,12 @@
                     "typeName": "Image",
                     "icon": "image outline",
                     "defaultName": "myImage"
+                },
+                {
+                    "label": "Location",
+                    "typeName": "tiles.Location",
+                    "icon": "tree",
+                    "defaultName": "myLocation"
```

Comment:
I know there's a bit of a convention that we have for variable names, but to me, myLocation feels a bit nebulous. Could it be `tileLocation` instead or is that too long? It also might be because I don't work with tilemaps much and can be disregarded.

Reply:
this is what we use in the scene category, so i'm gonna keep it